.lintstagedrc.yaml

```diff
@@ -0,0 +1,2 @@
+package.json: yarn sort-package-json
+'*.{json,md,yaml}': yarn prettier --write
```

Comment:
might need to double-check if these will overlap with package-specific ones causing a race condition https://github.com/okonet/lint-staged#filtering-files

Reply:
Running `lint-staged -v -d` and review the result. This works as expected. No overlap.